compiler/record-hessian/src/HessianComputer.cpp

```diff
@@ -130,17 +124,17 @@ void HessianComputer::recordHessianForConv2D(const luci::CircleNode *node,
 {
   assert(input_tensor->shape().rank() == 4);
   assert(input_tensor->element_type() == DataType::FLOAT32);
-  const auto circle_conv2d = (luci::CircleConv2D *)node;
-  const auto node_filter = loco::must_cast<luci::CircleConst *>(
-    loco::must_cast<const luci::CircleConv2D *>(circle_conv2d)->filter());
-  const auto node_bias = loco::must_cast<luci::CircleConst *>(
-    loco::must_cast<const luci::CircleConv2D *>(circle_conv2d)->bias());
+
+  const auto circle_conv2d = loco::must_cast<const luci::CircleConv2D *>(node);
+  const auto node_filter = loco::must_cast<luci::CircleConst *>((circle_conv2d)->filter());
+  const auto node_bias = loco::must_cast<luci::CircleConst *>((circle_conv2d)->bias());
```

Comment:
`bias` can be `luci::CircleOutputExclude` if Conv2D has no bias.
You need to handle this.